### ISSUE TITLE
Too low contrast on A/B test colour scheme

### ISSUE THREAD

**Issue by abimysoc** [mysociety/fixmystreet#1357]
User feedback:

> The colour contrasts on the new top menu bar make it difficult to
read for me, even though my vision is not particularly poor.  I
think it will be unreadable by someone with poor vision.

> I haven't checked it against any of the formal WCAG guidelines on
colour contrasts, but would be surprised if it passed.

I agree. This screenshot is from the admin interface:

![fms-ab](https://cloud.githubusercontent.com/assets/560001/13740198/5692ddd2-e9c7-11e5-9377-2e3cc6d18e4a.png)

**Comment by dracos** [comment]
Note the admin is lacking the background picture the front end has, which makes the contrast worse. I do agree it could perhaps be better anyway.

**Comment by zarino** [comment]
Yes, could deffo be better. Sorry for shipping it as-is, not sure what I was thinking. As discussed with @dracos, putting the text in black (and removing the text-shadow) is probably the best solution. Happy for @dracos to do this – just make a judgement call on the homepage as to whether the nav links there should stay white with a drop-shadow, or go black to match the other pages.
